explanations/best-practices/code-reviews.rst

```diff
@@ -0,0 +1,37 @@
+============
+Code reviews
+============
+
+Writing code is a craft. There is no absolute right or wrong way of expressing thoughts into code. It's mainly contextual:
+
+- What are the current needs?
+- What are the current constraints?
+- What is our current understanding of the problem?
+
+Reviewing code is also contextual. Here below are some guidelines. They must not be applied in every circumstance. Instead, they define a baseline in terms of expectations on the team.
+
+Requesting a review
+===================
+
+- **Load:** requests should be well balanced among team members so we don't overload a single individual. GitHub's CODEOWNERS files can help with this.
+- **Reminders:** the Slack channel `#releng-notifications <https://app.slack.com/client/T027LFU12/CN77C0BUG>`__ will remind the reviewer when there is an outstanding request.
```

Comment:
In my experience that doesn't actually happen in most cases, e.g. I don't remember seeing reminders there for mozilla-releng projects or for phabricator revisions.  Saying "will remind the reviewer" makes it sound a lot more definitive.

Reply:
Good point. Let's make this point more actionable then and allow the requester to remind the reviewer after 2 business days.